probe_scraper/parsers/third_party/histogram_tools.py

```diff
@@ -76,7 +76,7 @@ def exponential_buckets(dmin, dmax, n_buckets):
 
 always_allowed_keys = ['kind', 'description', 'cpp_guard', 'expires_in_version',
                        'alert_emails', 'keyed', 'releaseChannelCollection',
-                       'bug_numbers', 'record_in_processes']
+                       'bug_numbers', 'record_in_processes', 'record_into_store']
```

Comment:
probe-scraper takes the tactic of editing histogram_tools instead of pulling in newer copies of mozparsers?

Reply:
So I first assumed it uses the proper mozparsers, but then I diffed against our versions and they were vastly apart, so I didn't bother trying to use them.